modules/hosted-che/partials/ref_about-hosted-che.adoc

```diff
@@ -17,9 +17,15 @@ NOTE: link:https://workspaces.openshift.com/[Eclipse Che hosted by Red Hat] prov
 
 image::hosted-che/get-started-product-and-community-devfiles.png[]
 
-WARNING: Community-supported devfile registry has been temporarily disabled on the link:https://workspaces.openshift.com/[Eclipse Che hosted by Red Hat] due to a defect in the product. It is going to be re-enabled with the new release in September 2021.
-
-IMPORTANT: Eclipse Che and Red Hat CodeReady Workspaces share all of the features - all the product's functionality is available in the project and vice versa. However, not all the upstream plugins are available in the CodeReady Workspaces. To use an unsupported plugin inside the CodeReady Workspaces, one needs to explicitly point to the raw `meta.yaml` of the plugin from the devfile. The procedure is described in the xref:end-user-guide:adding-a-vs-code-extension-to-a-workspace.adoc#adding-the-vs-code-extension-using-the-workspace-configuration_che[Adding a VS Code extension using the workspace configuration] section.
+[WARNING]
+====
+The community-supported devfile registry has been temporarily disabled on the link:https://workspaces.openshift.com/[Eclipse Che hosted by Red Hat] due to a defect in the product. This devfile registry will be re-enabled for Hosted Che with the upcoming release in September 2021.
+====
+
+[IMPORTANT]
+====
+Eclipse Che and Red Hat CodeReady Workspaces share all of the features - all the product's functionality is available in the project and vice versa. However, not all the upstream plugins are available in the CodeReady Workspaces. To use an unsupported plugin inside the CodeReady Workspaces, one must explicitly point to the raw `meta.yaml` of the plugin from the devfile. The procedure is described in the xref:end-user-guide:adding-a-vs-code-extension-to-a-workspace.adoc#adding-the-vs-code-extension-using-the-workspace-configuration_che[Adding a VS Code extension using the workspace configuration] section.
+====
```

Comment:
@MichalMaler could you clarify why this formatting is obligatory?

Reply:
@ibuziuk There is an "OK" formatting and the correct formatting. I apply the correct formatting for all additions that have more that one sentence.

Reply:
that is fine, just wondering where this guidance is defined?

Reply:
Here is the definition:
https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/
The block is applied when the content is on more than one raw. Also, by doing it that way, it is possible to add additional markup aspects, such as a bullet or ordered lists, to the content. In this case, it was my personal preference. Your notation was OK.